packages/styleguide/stories/Foundations/Colors/styles.module.scss

```
@@ -35,19 +35,47 @@ h2.headingDark {
 }
 
 .swatch {
-  height: 64px;
-  width: 104px;
+  position: relative;
+  display: flex;
+  height: 125px;
+  width: 100%;
+  max-width: 100%;
   border: 1px solid $color-gray-800;
   border-bottom: 0;
-  margin-right: 8px;
+}
+
+.swatchButton {
+  position: relative;
+  font-weight: 700;
+  font-size: 13px;
+  font-family: $font-family-headings;
+  opacity: 0;
+  transition: opacity 0.5s;
+}
+
+.swatchDetail {
+  height: 100%;
+  &:hover {
```

This way we can <kbd>Tab</kbd> through them:

```suggestion
  &:focus,
  &:hover {
```